plugin/src/main/java/net/dzikoysk/funnyguilds/config/MessageConfiguration.java

```diff
@@ -362,14 +362,16 @@ public class MessageConfiguration extends OkaeriConfig {
     public String inviteAllyAmount = "&7Osiagnieto juz &cmaksymalna &7liczbe sojuszy miedzygildyjnych! (&c{AMOUNT}&7)";
     @Comment("Dostępne zmienne: {AMOUNT}, {GUILD}, {TAG}")
     public String inviteAllyTargetAmount = "&7Gildia {TAG} posiada juz maksymalna liczbe sojuszy! (&c{AMOUNT}&7)";
-    public String inviteCancelled = "&cCofnieto zaproszenie!";
+    public String inviteCancelled = "&cCofnieto zaproszenie dla {USER}!";
```

Comment:
Unrelated do PR. 😂

Reply:
Related - chyba nie chcesz dostać pięć wiadomośći, że zaproszono gracza do gildii. Wolisz wiedzieć kogo zapraszasz

Reply:
To jest tak naprawdę coś, co powinno być dla mnie przed tym PR włączone, ale ujdzie.